qabelbox/src/main/java/de/qabel/desktop/repository/sqlite/SqliteContactRepository.java

```diff
@@ -195,4 +204,71 @@ public synchronized Contact findByKeyId(Identity identity, String keyId) throws
             throw new EntityNotFoundExcepion("exception while searching contact: " + e.getMessage(), e);
         }
     }
+
+    @Override
+    public Collection<Pair<Contact, List<Identity>>> find(Identities identities, String searchString) throws PersistenceException {
```

Comment:
The searchString is apparently nullable. Splitting this up into different methods, one without the searchString and one with a non-nullable searchString could make this method easier to grasp.